src/app/ReadClient.cpp

```diff
@@ -541,9 +541,18 @@ CHIP_ERROR ReadClient::ProcessEventReportIBs(TLV::TLVReader & aEventReportIBsRea
 CHIP_ERROR ReadClient::RefreshLivenessCheckTimer()
 {
     CancelLivenessCheckTimer();
+    System::Clock::Timeout timeout =
+        std::chrono::duration_cast<std::chrono::milliseconds>(System::Clock::Seconds16(mMaxIntervalCeilingSeconds));
+    if (mpExchangeCtx != nullptr && mpExchangeCtx->IsUDPTransport())
+    {
+        System::Clock::Timeout margin = System::Clock::Milliseconds32((CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS + 1) *
+                                                                      mpExchangeCtx->GetIdleRetransmitTimeoutTick());
```

Comment:
`mpExchangeCtx->GetIdleRetransmitTimeoutTick()` does not return a number of milliseconds.  It returns a number of ticks.  Ticks are not one millisecond.

Reply:
yep, looks like mpExchangeCtx->GetIdleRetransmitTimeoutTick() >> CHIP_CONFIG_RMP_TIMER_DEFAULT_PERIOD_SHIFT which can return a number of msec